libs/dh/shared/data-access-mocks/src/lib/data/metering-point/child-metering-point.ts

```diff
@@ -37,6 +37,7 @@ import { parentMeteringPoint } from './parent-metering-point';
 const metadata: MeteringPointMetadataDto = {
   __typename: 'MeteringPointMetadataDto',
   id: '444444411',
+  parentMeteringPoint: '222222211',
```

Comment:
suggestion: let's use this value to keep it the same as before

```suggestion
  parentMeteringPoint: '222222222222222222',
```